code/go/0chain.net/blobbercore/handler/storage_handler.go

```diff
@@ -26,6 +26,10 @@ const (
 
 	DOWNLOAD_CONTENT_FULL  = "full"
 	DOWNLOAD_CONTENT_THUMB = "thumbnail"
+	PAGE_LIMIT             = 100 //100 rows will make upto 100 KB
```

Comment:
Use Camel Case naming

Reply:
Done

Reply:
> I don't have context on 0fs and the pagination followed here, according to the original issue the need was to have `/v1/file/list/{allocation}` accept query params & support pagination. Since I am still new to the project, I have just pointed out the general golang improvements.

Its actually replacement to `/v1/file/objectpath/{allocation}`, ` /v1/file/referencepath/{allocation}` and  `/v1/file/objecttree/{allocation}`
But it can be used widely by any use cases where it is required to fetch large trees.

This allows client application to build any kind/size of tree rather than overloading blobber.